### ISSUE TITLE
Fetch Realtime Data and Bar Data (historical) from Interactive Brokers using Stoqey Node-IB Library

### ISSUE THREAD

**Issue by drixie** [drixie/royaloak#4]
1. The end goal is to use this data for (a) calculating indicator values, (b) executing order rules for assets
2. Calculated indicator data should be saved in the database for each asset (Idris to show Fabio how to use existing python code for indicator calculations... there are no reliable nodejs libraries for indicator calculations)
3. The data should be fetched for all assets saved in the watchlist

Notes: 
a) This data will eventually be used to display charts and levels in the TradingView Library

**Comment by drixie** [comment]
@rylorin See this issue for the data format compatible with TradingView charting library - https://github.com/drixie/royaloak/issues/9

**Comment by rylorin** [comment]
The mentioned issue is related to symbol info, not to data format ...
And I suggest to split this issue in 2 parts: 1 part for realtime data and 1 part for historical data, as they probably have different usages.

**Comment by drixie** [comment]
@rylorin I agree with you on the point about splitting. Actually, I think it should be split into four parts: 
1. Fetch data relevant for symbolinfo (I think this should be a big list of IB contracts relevant to all US equities) - https://github.com/drixie/royaloak/issues/10
2. Fetch Realtime Data - Tick by Tick Data (for Time and Sales Window) - https://github.com/drixie/royaloak/issues/11
3. Fetch Realtime Data - Snapshots (For TradingView charts because TradingView does not accept tick by tick data) - https://github.com/drixie/royaloak/issues/12
4. Fetch Historical Data (for calculating Indicator values used for setting Levels and Executing Rules) - https://github.com/drixie/royaloak/issues/13

**Comment by drixie** [comment]
I have created new issues for the four areas indicated above. I will close this issue if there are no other other comments
